### DISCUSSION TITLE
plot a dataset coordinate only

### DISCUSSION THREAD

**Discussion by michaelaye** [pydata/xarray#6048] [Q&A]
For checking the distribution over a specific coordinate, it would be good to make it easy to plot any given coordinate over its index, like what happens when one does in MPL `plt.plot(arr)` where the x-axis is automatically set as the index.

I have this dataset:

![image](https://user-images.githubusercontent.com/69774/144729463-0f035cb3-7806-4fd5-a736-a1665eff4751.png)

and when I want to do the obvious (TM ;) ) of 

`ds.H.hvplot()`

I get:

```
ValueError: cannot create a Dataset from a DataArray with the same name as one of its coordinates
```

What's the best practice to be able to quickly plot any given coordinate of a dataset?

**Comment by Illviljan** [comment]
For hvplot errors  https://github.com/holoviz/hvplot can probably help better, but just using `plt.plot` seems quick enough:
```python
ds = xr.tutorial.scatter_example_dataset()
plt.plot(ds.w)
```
![image](https://user-images.githubusercontent.com/14371165/144904175-bd49e94a-2c87-45a3-84d2-1ebf432912b6.png)

Using `.plot()` doesn't crash at least although I do question how useful it is to plot `w` along `w`:
```python
ds.w.plot()
```
![image](https://user-images.githubusercontent.com/14371165/144904354-920b7203-800b-48b3-9fe5-b8871671b7e0.png)

**Comment by andhuang-CLGX** [comment]
Something like:
`ds.reset_coords().H.hvplot()` maybe
